armory/analyzer/driver.cc

```diff
@@ -72,6 +72,7 @@ static void PopulatePassesAndRun(GlobalContext& ctx, Module& m,
   }
   pm.AddPass<DCE>();
   pm.AddPass<TypeLegalizer>(true);
+  pm.AddPass<InstSimplify>(true, true, false, false, false, false);
   auto analyzer = pm.AddPass<Analyzer>();
```

Comment:
maybe we can pass a structure to InstSimplify pass， such as `struct SimplifyOption` to improve readability